app/src/main/res/layout/activity_main.xml

```diff
@@ -1,22 +1,26 @@
 <?xml version="1.0" encoding="utf-8"?>
-<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
I'd continue using the ConstraintLayout, it's an interesting learning experience. Seems complex at first, but when you get how it works, it's really cool